### ISSUE TITLE
Start Work: Update Jira issue search request to sort by update date

### ISSUE THREAD

**Issue by sergeibbb** [gitkraken/vscode-gitlens#3740]
This issue is a follow-up of #3621

## The problem

For Jira, on the _Start Work_ view I'm only seeing issues that I created/authored. It should show, and prioritize, issues I am assignee on, [like this one for example](https://gitkraken.atlassian.net/browse/GLVSC-427). Instead I am seeing issues I reported even if they have no assignee.

#### The explanation
The mentioned issue probably is in the search result, because the current search query is:
```
(creator = currentUser() OR assignee = currentUser() OR comment ~ currentUser() OR summary ~ currentUser() OR description ~ currentUser())
```

but in the current order it doesn't appear on the first page, that's why the user doesn't see it.

## The ~~solution~~ proposal of the solution
https://github.com/gitkraken/provider-apis-package-js/pull/194

Require issues to be ordered by update date. So the user will the recently updated items first and most likely see the issues they are interested in. Stale items will be at the bottom of the list or cut out by the query limit.

Add the following suffix to [the query](https://github.com/gitkraken/provider-apis-package-js/blob/262b128b15868d9cfd850480aa54cc091c23d70b/src/providers/jira/jiraHelpers.ts#L252-L256):
```
ORDER BY updated DESC
```

_This solution is questionable, it has its advantages and disadvantages._

Advantages:
- it's easy to do
Disadvantages:
- _"The most recently updated issues where the user has been involved"_ is not the same as "the issues the user is most involved in". Probably a better solution would be somehow to order by a date of the most recent change related to the user. But there isn't a way to achieve it in JQL.

**Comment by sergeibbb** [comment]
@d13 , @axosoft-ramint 

[The library's PR](https://github.com/gitkraken/provider-apis-package-js/pull/194) has been merged.
As soon as they publish it in NPM with a version, the updating of the dependency in our code will be the solution of the current ticket.

cc @justinrobots

**Comment by gitkraken-jacobw** [comment]
@d13, @axosoft-ramint

It looks like that Sergei is off this week, so I'm posting this in his stead.

I just deployed v0.25.5 of the provider library, which includes Sergei's PR. All that you guys need to do is bump the library to v0.25.5 and this issue should be resolved.

cc @justinrobots

**Comment by sergeibbb** [comment]
Fixed in eec532fb2fa6027a33db1326a2be57efd34adce6 (#3719)
